<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="6" failed="0" total="6" passed="0">
  <reporter-output>
    <line>
      <![CDATA[[UserExtension][2016.05.04]'[03:01:57]     Fail: Unable to create the RemotedriverCould not start a new session. Possible causes are invalid address of the remote server or browser start-up failure.
Build info: version: '2.46.0', revision: '87c69e2', time: '2015-06-04 16:17:10'
System info: host: 'G-0439', ip: '10.163.105.61', os.name: 'Windows 7', os.arch: 'x86', os.version: '6.1', java.version: '1.7.0_80'
Driver info: driver.version: RemoteWebDriver]]>
    </line>
  </reporter-output>
  <suite name="SRM" duration-ms="3068" started-at="2016-05-04T09:31:54Z" finished-at="2016-05-04T09:31:58Z">
    <groups>
    </groups>
    <test name="Orion3_Navigation_FireFox" duration-ms="3068" started-at="2016-05-04T09:31:54Z" finished-at="2016-05-04T09:31:58Z">
      <class name="validateCollege_InformationSRM.ValidateUpdatedCollege_Information">
        <test-method status="FAIL" signature="BeforeNavigation(java.lang.String)[pri:0, instance:validateCollege_InformationSRM.ValidateUpdatedCollege_Information@1025f9f]" name="BeforeNavigation" is-config="true" duration-ms="0" started-at="2016-05-04T15:01:54Z" finished-at="2016-05-04T15:01:54Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[firefox]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
	at validateCollege_InformationSRM.ValidateUpdatedCollege_Information.BeforeNavigation(ValidateUpdatedCollege_Information.java:180)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:515)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:213)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:140)
	at org.testng.internal.TestMethodWorker.invokeBeforeClassMethods(TestMethodWorker.java:170)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:104)
	at org.testng.TestRunner.privateRun(TestRunner.java:771)
	at org.testng.TestRunner.run(TestRunner.java:621)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1176)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1101)
	at org.testng.TestNG.run(TestNG.java:1009)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
            <line>
              <![CDATA[[UserExtension][2016.05.04]'[03:01:57]     Fail: Unable to create the RemotedriverCould not start a new session. Possible causes are invalid address of the remote server or browser start-up failure.
Build info: version: '2.46.0', revision: '87c69e2', time: '2015-06-04 16:17:10'
System info: host: 'G-0439', ip: '10.163.105.61', os.name: 'Windows 7', os.arch: 'x86', os.version: '6.1', java.version: '1.7.0_80'
Driver info: driver.version: RemoteWebDriver]]>
            </line>
          </reporter-output>
        </test-method> <!-- BeforeNavigation -->
        <test-method status="SKIP" signature="BrowseToAddInfoCallLeadPage(java.lang.reflect.Method)[pri:0, instance:validateCollege_InformationSRM.ValidateUpdatedCollege_Information@1025f9f]" name="BrowseToAddInfoCallLeadPage" duration-ms="5" started-at="2016-05-04T15:01:58Z" finished-at="2016-05-04T15:01:58Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void validateCollege_InformationSRM.ValidateUpdatedCollege_Information.BrowseToAddInfoCallLeadPage(java.lang.reflect.Method) throws java.lang.InterruptedException]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- BrowseToAddInfoCallLeadPage -->
        <test-method status="SKIP" signature="VerifyLeadInSRM(java.lang.reflect.Method)[pri:0, instance:validateCollege_InformationSRM.ValidateUpdatedCollege_Information@1025f9f]" name="VerifyLeadInSRM" duration-ms="0" started-at="2016-05-04T15:01:58Z" depends-on-methods="validateCollege_InformationSRM.ValidateUpdatedCollege_Information.BrowseToAddInfoCallLeadPage" finished-at="2016-05-04T15:01:58Z">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method ValidateUpdatedCollege_Information.VerifyLeadInSRM(java.lang.reflect.Method)[pri:0, instance:validateCollege_InformationSRM.ValidateUpdatedCollege_Information@1025f9f] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method ValidateUpdatedCollege_Information.VerifyLeadInSRM(java.lang.reflect.Method)[pri:0, instance:validateCollege_InformationSRM.ValidateUpdatedCollege_Information@1025f9f] depends on not successfully finished methods
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1052)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:124)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:771)
	at org.testng.TestRunner.run(TestRunner.java:621)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1176)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1101)
	at org.testng.TestNG.run(TestNG.java:1009)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- VerifyLeadInSRM -->
        <test-method status="SKIP" signature="Lead_Flow(java.lang.reflect.Method)[pri:0, instance:validateCollege_InformationSRM.ValidateUpdatedCollege_Information@1025f9f]" name="Lead_Flow" duration-ms="0" started-at="2016-05-04T15:01:58Z" depends-on-methods="validateCollege_InformationSRM.ValidateUpdatedCollege_Information.VerifyLeadInSRM" finished-at="2016-05-04T15:01:58Z">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method ValidateUpdatedCollege_Information.Lead_Flow(java.lang.reflect.Method)[pri:0, instance:validateCollege_InformationSRM.ValidateUpdatedCollege_Information@1025f9f] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method ValidateUpdatedCollege_Information.Lead_Flow(java.lang.reflect.Method)[pri:0, instance:validateCollege_InformationSRM.ValidateUpdatedCollege_Information@1025f9f] depends on not successfully finished methods
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1052)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:124)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:771)
	at org.testng.TestRunner.run(TestRunner.java:621)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1176)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1101)
	at org.testng.TestNG.run(TestNG.java:1009)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Lead_Flow -->
        <test-method status="SKIP" signature="SRM__Lead_Details(java.lang.reflect.Method)[pri:0, instance:validateCollege_InformationSRM.ValidateUpdatedCollege_Information@1025f9f]" name="SRM__Lead_Details" duration-ms="0" started-at="2016-05-04T15:01:58Z" depends-on-methods="validateCollege_InformationSRM.ValidateUpdatedCollege_Information.Lead_Flow" finished-at="2016-05-04T15:01:58Z">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method ValidateUpdatedCollege_Information.SRM__Lead_Details(java.lang.reflect.Method)[pri:0, instance:validateCollege_InformationSRM.ValidateUpdatedCollege_Information@1025f9f] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method ValidateUpdatedCollege_Information.SRM__Lead_Details(java.lang.reflect.Method)[pri:0, instance:validateCollege_InformationSRM.ValidateUpdatedCollege_Information@1025f9f] depends on not successfully finished methods
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1052)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:124)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:771)
	at org.testng.TestRunner.run(TestRunner.java:621)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1176)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1101)
	at org.testng.TestNG.run(TestNG.java:1009)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- SRM__Lead_Details -->
        <test-method status="SKIP" signature="VerifyOpportunityInSRM(java.lang.reflect.Method)[pri:0, instance:validateCollege_InformationSRM.ValidateUpdatedCollege_Information@1025f9f]" name="VerifyOpportunityInSRM" duration-ms="0" started-at="2016-05-04T15:01:58Z" depends-on-methods="validateCollege_InformationSRM.ValidateUpdatedCollege_Information.SRM__Lead_Details" finished-at="2016-05-04T15:01:58Z">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method ValidateUpdatedCollege_Information.VerifyOpportunityInSRM(java.lang.reflect.Method)[pri:0, instance:validateCollege_InformationSRM.ValidateUpdatedCollege_Information@1025f9f] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method ValidateUpdatedCollege_Information.VerifyOpportunityInSRM(java.lang.reflect.Method)[pri:0, instance:validateCollege_InformationSRM.ValidateUpdatedCollege_Information@1025f9f] depends on not successfully finished methods
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1052)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:124)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:771)
	at org.testng.TestRunner.run(TestRunner.java:621)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1176)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1101)
	at org.testng.TestNG.run(TestNG.java:1009)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- VerifyOpportunityInSRM -->
        <test-method status="SKIP" signature="VerifyNewOpportunityInSRM(java.lang.reflect.Method)[pri:0, instance:validateCollege_InformationSRM.ValidateUpdatedCollege_Information@1025f9f]" name="VerifyNewOpportunityInSRM" duration-ms="0" started-at="2016-05-04T15:01:58Z" depends-on-methods="validateCollege_InformationSRM.ValidateUpdatedCollege_Information.VerifyOpportunityInSRM" finished-at="2016-05-04T15:01:58Z">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method ValidateUpdatedCollege_Information.VerifyNewOpportunityInSRM(java.lang.reflect.Method)[pri:0, instance:validateCollege_InformationSRM.ValidateUpdatedCollege_Information@1025f9f] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method ValidateUpdatedCollege_Information.VerifyNewOpportunityInSRM(java.lang.reflect.Method)[pri:0, instance:validateCollege_InformationSRM.ValidateUpdatedCollege_Information@1025f9f] depends on not successfully finished methods
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1052)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:124)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:771)
	at org.testng.TestRunner.run(TestRunner.java:621)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1176)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1101)
	at org.testng.TestNG.run(TestNG.java:1009)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- VerifyNewOpportunityInSRM -->
        <test-method status="SKIP" signature="AfterNavigation()[pri:0, instance:validateCollege_InformationSRM.ValidateUpdatedCollege_Information@1025f9f]" name="AfterNavigation" is-config="true" duration-ms="0" started-at="2016-05-04T15:01:58Z" finished-at="2016-05-04T15:01:58Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- AfterNavigation -->
      </class> <!-- validateCollege_InformationSRM.ValidateUpdatedCollege_Information -->
      <class name="srm_Variables.EnvironmentVariables">
        <test-method status="PASS" signature="BeforeSuite_Orion3(java.lang.String, java.lang.String, java.lang.String, org.testng.ITestContext)[pri:0, instance:srm_Variables.EnvironmentVariables@4434f7]" name="BeforeSuite_Orion3" is-config="true" duration-ms="524" started-at="2016-05-04T15:01:54Z" finished-at="2016-05-04T15:01:54Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[STAGE]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[localhost]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[4444]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[org.testng.TestRunner@14bee22]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- BeforeSuite_Orion3 -->
      </class> <!-- srm_Variables.EnvironmentVariables -->
    </test> <!-- Orion3_Navigation_FireFox -->
  </suite> <!-- SRM -->
</testng-results>
